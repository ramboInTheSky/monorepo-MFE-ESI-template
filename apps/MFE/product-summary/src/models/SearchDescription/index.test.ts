import {EnableSearchDescription} from "."

describe("Model - EnableSearchDescription: ", () => {
    const mockTestState: EnableSearchDescription = {
        Value: false,
    }
    it("should match the EnableSearchDescription", () => {
        expect(mockTestState).toMatchSnapshot()
    })
})
