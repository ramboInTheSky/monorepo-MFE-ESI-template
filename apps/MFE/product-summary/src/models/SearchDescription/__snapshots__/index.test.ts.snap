// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Model - EnableSearchDescription:  should match the EnableSearchDescription 1`] = `
Object {
  "Value": false,
}
`;
